bin/utils/subkey/src/main.rs

```diff
@@ -329,13 +398,24 @@ where
 	if let Some(network) = maybe_network {
 		set_default_ss58_version(network);
 	}
+
+	let maybe_output: Option<OutputType> = match matches.value_of("output").map(|output| {
```

Comment:
should not be optional?
If there is none in matches provided, default one is instantiated